### ISSUE TITLE
Incorrect description in header component

### ISSUE THREAD

**Issue by edwardhorsford** [alphagov/govuk-design-system#835]
The header component Nunjucks macro options description has the same text repeated for `productName` and `serviceName` - I suspect service name is meant to have text talking about common service names?

![Screenshot 2019-03-21 at 10 07 21](https://user-images.githubusercontent.com/2204224/54745395-50548b00-4bc1-11e9-97f6-04e8348f7829.png)

**Comment by dashouse** [comment]
@hannalaakso Are these populated from the .yaml file in GOV.UK Frontend (https://github.com/alphagov/govuk-frontend/blob/5dc0c1c6e7af106ebab2ee68cf674032dc9b5304/src/components/header/header.yaml#L17)?

**Comment by hannalaakso** [comment]
Yes that's right, we just need to update what `serviceName` ought to communicate there 👍

**Comment by 36degrees** [comment]
Going to move this to the Frontend repo as that's where the fix needs to be made.
